optifit backend/pose_server.py

```
@@ -0,0 +1,315 @@
+            if landmarks_data:
+                response['landmarks'] = landmarks_data
+            
+            return jsonify(response)
+        else:
+            return jsonify({
+                'success': True,
+                'pose_detected': False,
+                'message': 'No pose detected in frame'
+            })
+            
+    except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)